blazor/datepicker/style-appearance.md

```diff
@@ -49,24 +49,19 @@ You can customize the background color of the container element for the DatePick
 
 ![DatePicker with background color](./images/blazor-datepicker-background_color.png)
 
-## Customizing the Calendar popup of the DatePicker
+## Full Screen Mode: Enhancing the DatePicker Component (Mobile Support Only)
 
-### Mobile mode full screen calendar popup in material theme
+The full-screen mode feature in the DatePicker component allows you to view the DatePicker popup element in full-screen mode on mobile devices, enabling you to see the DatePicker calendar element with better visibility and an upgraded user experience. It is important to mention that this feature is exclusively available for mobile devices in both landscape and portrait orientations. To activate the full screen mode within the DatePicker component, simply set the [FullScreenMode](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Calendars.SfDatePicker-1.html#Syncfusion_Blazor_Calendars_SfDatePicker_1_FullScreenMode) API value to `true`. This action will extend the calendar element to occupy the entire screen on mobile devices.
```

Comment:
The full-screen mode feature in the DatePicker component allows you to view the DatePicker popup element in full-screen on mobile devices, enabling you to see the DatePicker calendar element with better visibility and an upgraded user experience. It is important to mention that this feature is exclusively available for mobile and tablet devices in both landscape and portrait orientations. To activate the full screen mode within the DatePicker component, simply set the [FullScreenMode](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Calendars.SfDatePicker-1.html#Syncfusion_Blazor_Calendars_SfDatePicker_1_FullScreenMode) API value to `true`. This action will extend the calendar element to occupy the entire screen on mobile and tablet devices.

Reply:
Update the above changes in all platforms